Kitodo/src/main/java/org/kitodo/services/data/base/TitleSearchService.java

```diff
@@ -28,10 +30,11 @@
  * Class for implementing methods used by service classes which search for title
  * in ElasticSearch index.
  */
-public abstract class TitleSearchService<T extends BaseIndexedBean, S extends BaseDTO, V extends BaseDAO<T>> extends SearchService<T, S, V> {
+public abstract class TitleSearchService<T extends BaseIndexedBean, S extends BaseDTO, V extends BaseDAO<T>>
+        extends SearchService<T, S, V> {
 
-    public TitleSearchService(V dao, Searcher searcher) {
-        super(dao, searcher);
+    public TitleSearchService(V dao, BaseType type, Indexer indexer, Searcher searcher) {
```

Comment:
Missing JavaDoc for class constructor or add a override annotation.

Reply:
done